examples/transformers/v2-protocol/README.ipynb

```diff
@@ -253,14 +253,7 @@
     "                securityContext:\n",
     "                  # Workaround for multi-user bug in custom images in MLServer:\n",
     "                  # https://github.com/SeldonIO/MLServer/issues/388\n",
-    "                  runAsUser: 1000\n",
-    "                ports:\n",
-    "                  - containerPort: 8080\n",
-    "                    name: http\n",
-    "                    protocol: TCP\n",
-    "                  - containerPort: 8081\n",
-    "                    name: grpc\n",
-    "                    protocol: TCP\n"
```

Comment:
That's a good point. I think that, in that case, the operator would just pick those values (rather than generating new ones), and those would then get injected in MLServer through env vars.